README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+##### HOMEWORK ASSIGNMENT 1
+I'm **Jiayang He**, a MAPSS student concentrating on *psychology*.
+
+My interest in the area focuses on several specific topics under certain sub-fields of psychology:
+
+* Psychophysiology
+  + Will stress be contagious via watching stressful videos? 
+* Social neuronscience
+  + How social stress impact interoception?
+* Clinical psychology
+  + Will people with depresssion or panic disorder have a different pattern of stress response?
+
+I am currently working in Dr. Norman's SPNL (Social Psychophysiology & Neuroendocrinology Lab). Here's our website: <https://sites.google.com/a/uchicago.edu/spnl/>.
```

Comment:
Better use of Markdown to format links

```
I am currently working in Dr. Norman's [SPNL (Social Psychophysiology & Neuroendocrinology Lab)](https://sites.google.com/a/uchicago.edu/spnl/).
```

I am currently working in Dr. Norman's [SPNL (Social Psychophysiology & Neuroendocrinology Lab)](https://sites.google.com/a/uchicago.edu/spnl/).